src/api/cvc4cpp.cpp

```diff
@@ -253,10 +253,10 @@ const static std::unordered_map<Kind, CVC4::Kind, KindHashFunction> s_kinds{
     {STRING_LENGTH, CVC4::Kind::STRING_LENGTH},
     {STRING_SUBSTR, CVC4::Kind::STRING_SUBSTR},
     {STRING_CHARAT, CVC4::Kind::STRING_CHARAT},
-    {STRING_STRCTN, CVC4::Kind::STRING_STRCTN},
-    {STRING_STRIDOF, CVC4::Kind::STRING_STRIDOF},
-    {STRING_STRREPL, CVC4::Kind::STRING_STRREPL},
-    {STRING_STRREPLALL, CVC4::Kind::STRING_STRREPLALL},
+    {STRING_CONTAINS, CVC4::Kind::STRING_STRCTN},
+    {STRING_INDEXOF, CVC4::Kind::STRING_STRIDOF},
+    {STRING_REPLACE, CVC4::Kind::STRING_STRREPL},
+    {STRING_REPLACEALL, CVC4::Kind::STRING_STRREPLALL},
```

Comment:
```suggestion
    {STRING_REPLACE_ALL, CVC4::Kind::STRING_STRREPLALL},
```

(the symbol name is `str.replace_all`)